src/openzaak/components/documenten/query/cmis.py

```diff
@@ -773,6 +774,7 @@ def process_filters(self, data):
                     list_value.append(make_absolute_uri(reverse(item)))
                 value = list_value
 
+            # FIXME
```

Comment:
should we create a ticket for this?

Reply:
Not necessary, I think I added this at the time because I wasn't sure if it would work, but because `value` is always a URL in the case of verzoeken, the `else` statement is fine for this case